### PR TITLE
Fix `value-keyword-case` false positives for vendor-prefixed system colors

### PR DESCRIPTION
**Pull request by Mouvedia**
> Which issue, if any, is this issue related to?

#1957

> Is there anything in the PR that needs further explanation?

- `-ms-`: https://adrianroselli.com/2021/02/whcm-and-system-colors.html#CSS2WHCM
- `-moz-`: https://developer.mozilla.org/en-US/docs/Web/CSS/Mozilla_Extensions#color_keywords

---

I didn't add non-system colors.
e.g. `-moz-activehyperlinktext`, `-moz-default-background-color`, `-moz-default-color`, `-moz-hyperlinktext`, `-moz-visitedhyperlinktext`, etc.

### CODE REVIEW

File: lib/reference/keywords.mjs [hunk 1]
```diff
@@ -296,7 +296,49 @@ export const namedTimelineRangeKeywords = new Set([
 	'exit-crossing',
 ]);
 
-export const systemColorsKeywords = new Set([
+const prefixedSystemColorKeywords = new Set([
```
Comment (ybiquitous) [inline lib/reference/keywords.mjs:299]: [suggestion] Can you add a code comment with links to web pages to maintain this list?
E.g. https://developer.mozilla.org/en-US/docs/Web/CSS/Mozilla_Extensions#color_keywords

**Review by ybiquitous — APPROVED** [review]
Thanks, overall LGTM. I've left one suggestion.

File: lib/reference/keywords.cjs [hunk 2]
```diff
@@ -300,7 +300,49 @@ const namedTimelineRangeKeywords = new Set([
 	'exit-crossing',
 ]);
 
-const systemColorsKeywords = new Set([
+const prefixedSystemColorKeywords = new Set([
```
Comment (Mouvedia) [inline lib/reference/keywords.cjs:303]:
```suggestion
// https://developer.mozilla.org/en-US/docs/Web/CSS/Mozilla_Extensions#color_keywords
const prefixedSystemColorKeywords = new Set([
```

File: lib/reference/keywords.mjs (hunk 1 above)
Comment (Mouvedia) [reply lib/reference/keywords.mjs:299]:
```suggestion
// https://developer.mozilla.org/en-US/docs/Web/CSS/Mozilla_Extensions#color_keywords
const prefixedSystemColorKeywords = new Set([
```

**Pull request merged**